#tr3:
  joint_limits:

    b0:
      has_position_limits: false
      has_velocity_limits: true
      min_velocity: -8.8
      max_velocity: 8.8
      has_acceleration_limits: false
      has_jerk_limits: false
      has_effort_limits: true
      max_effort: 100.0
      min_effort: -100.0

    b1:
      has_position_limits: false
      has_velocity_limits: true
      min_velocity: -8.8
      max_velocity: 8.8
      has_acceleration_limits: false
      has_jerk_limits: false
      has_effort_limits: true
      max_effort: 100.0
      min_effort: -100.0

    a0:
      has_position_limits: true
      min_position: -2.0
      max_position: 2.0
      has_velocity_limits: true
      max_velocity: 0.20
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 100.0
      min_effort: -100.0

    a1:
      has_position_limits: false
      has_velocity_limits: true
      max_velocity: 0.20
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 100.0
      min_effort: -100.0

    a2:
      has_position_limits: false
      has_velocity_limits: true
      max_velocity: 0.20
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 100.0
      min_effort: -100.0

    a3:
      has_position_limits: false
      has_velocity_limits: true
      max_velocity: 0.20
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 100.0
      min_effort: -100.0

    a4:
      has_position_limits: false
      has_velocity_limits: true
      max_velocity: 0.20
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 100.0
      min_effort: -100.0

    h0:
      has_position_limits: false
      has_velocity_limits: true
      max_velocity: 0.20
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 100.0
      min_effort: -100.0

    h1:
      has_position_limits: false
      has_velocity_limits: true
      max_velocity: 0.20
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 100.0
      min_effort: -100.0
